js/DataSourceGlobalFilter.js

```diff
@@ -41,6 +41,10 @@ var DataSourceGlobalFilter = DataSourceIndexed.extend('DataSourceGlobalFilter',
     get: function(filter) {
         return this.filter;
     },
+    
+    sortGroups: function(sorter){
+      this.dataSource.sortGroups(sorter);  
+    },
```

Comment:
Yea its being called. In the case where the filter takes a group sorter (i.e aggregations and group view)